WORKSPACE

```diff
@@ -35,9 +35,11 @@ bind(
 # When updating envoy sha manually please update the sha in istio.deps file also
 #
 # Determine SHA256 `wget https://github.com/envoyproxy/envoy/archive/COMMIT.tar.gz && sha256sum COMMIT.tar.gz`
-ENVOY_SHA = "5ea1a0c1cb506ed3e80d52b572b0f767f55f9f39"
+# envoy commit date  05/15/2019
+# bazel version: 0.25.0
```

Comment:
Nit: this isn't really true, since Bazel version depends on the build image / CI environment, and 0.25.x is only used on macOS right now, since it's pulled at build time.

Reply:
but it does work for 0.25... I think Yuchen wanted this so he could find a version that would work.

Reply:
Fair enough, but it uses 0.22 for Linux builds (CircleCI & Prow), 0.25 is only used for macOS.

Reply:
Let's merge this and I will create the correct a circleci image in which has bazel 0.25